fplll/nr/matrix.h

```diff
@@ -203,7 +203,7 @@ template <class T> class Matrix
   void print(ostream &os, int nrows = -1, int ncols = -1) const;
   /** Reads this matrix from a stream. */
   void read(istream &is);
-
+  /** Change the output format style of Matrix */
```

Comment:
This is me being really nit picky: but you sometimes use indicative ("changes") and sometimes imperative ("change"). We're already mixing those in the docs, but maybe it's worth to stick to one. I'd go with indicative here: https://carlomilanesi.wordpress.com/2012/02/09/grammar-style-for-code-comments/ Feel free to ignore this one, though, It is nit picky.